File: Napakalaki-Game(PDOO)/NapakalakiRuby/lib/prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.
#encoding: utf-8
class Prize
  attr_reader:treasures
  attr_reader:level
  
  def initialize(treasures,level)
    @treasures=treasures
    @level=level
  end
  
  
  
  def to_s
    "tesoros ganados: #{@treasures}\n Niveles Ganados: #{@level}"
  end
  
end

File: Napakalaki-Game(PDOO)/NapakalakiJava/src/napakalakiGame/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalakiGame;

/**
 * @author joaquin
 */
public class Prize {
    private int treasures;
    private int level;
    
    public Prize(int treasures, int level){
       this.treasures=treasures;
       this.level=level;
    
    }
    
    public int getTreasures(){
        
        
       return this.treasures;
    }
    
    public int getLevel(){
        
        
       return this.level;
    }

    public String toString(){
    
    return "\nTreasures = " + Integer.toString(treasures)+ " \nlevels = " + Integer.toString(level);
    }

}
